yt/frontends/gadget/simulation_handling.py

```diff
@@ -521,18 +521,18 @@ def _check_for_outputs(self, potential_outputs):
         for my_storage, output in parallel_objects(
             potential_outputs, storage=my_outputs
         ):
-            if os.path.exists(output):
-                try:
-                    ds = load(output)
-                    if ds is not None:
-                        my_storage.result = {
-                            "filename": output,
-                            "time": ds.current_time.in_units("s"),
-                        }
-                        if ds.cosmological_simulation:
-                            my_storage.result["redshift"] = ds.current_redshift
-                except YTOutputNotIdentified:
-                    mylog.error("Failed to load %s", output)
+            try:
```

Comment:
This whole method should be moved into the parent class, as it is used here and in `exodus_ii`.

Reply:
Sounds good to me, but would rather do this in a follow up PR if that's okay to you :-)

Reply:
That's fine for me.